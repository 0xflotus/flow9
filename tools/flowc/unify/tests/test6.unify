import tuples;
import maybe;
import behaviour;

// (DynamicBehaviour<Maybe<bool>>)⁺ c= (α7458)⁻   from   struct arg #1 in Triple() (Triple((hasSound : DynamicBehaviour<Maybe<bool>>), (sound : α7445), (soundLengt


DynamicBehaviour<Maybe<bool>> c= α7458;

α7449 c= .(.third : α7474);

α7474 c= Behaviour<α7473>;

// ((t : α7449) -> α7473)⁺ c= ((α7471) -> α7472)⁻   from   arg #2 in call of map() ((map : ([α7471], (α7471) -> α7472) -> [α7472])((soundsP : α7446), (\t:α7449 -> 
