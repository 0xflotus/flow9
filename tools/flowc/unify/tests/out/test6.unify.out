α7449=.(.third : {* .. Behaviour<α7473>})
α7458={DynamicBehaviour<Maybe<bool>> .. *}
α7474={* .. Behaviour<α7473>}
